xds/src/main/java/io/grpc/xds/XdsClient.java

```diff
@@ -100,6 +103,109 @@ ConfigUpdate build() {
     }
   }
 
+  static final class LdsUpdate {
+    // Total number of nanoseconds to keep alive an HTTP request/response stream.
+    private final long httpMaxStreamDuration;
+    // The name of the route configuration to be used for RDS resource discovery.
+    @Nullable
+    private final String rdsName;
+    // The list virtual hosts that make up the route table.
+    @Nullable
+    private final List<VirtualHost> virtualHosts;
+
+    private LdsUpdate(long httpMaxStreamDuration, @Nullable String rdsName,
+        @Nullable List<VirtualHost> virtualHosts) {
+      this.httpMaxStreamDuration = httpMaxStreamDuration;
+      this.rdsName = rdsName;
+      this.virtualHosts = virtualHosts;
+    }
+
+    long getHttpMaxStreamDuration() {
```

Comment:
Add javadoc saying it's in nanos or rename the method with `getHttpMaxStreamDurationNanos()`.

Reply:
Change to `getHttpMaxStreamDurationNanos()`. I don't think Javadoc is necessary as it's not a public API and it's just a getter. The method name tells everything.